mobile-app/components/Board.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import Svg from 'react-native-svg';
+import PlayableCell from './PlayableCell';
+import BorderCell from './BorderCell';
+import { GameState } from '../../web-app/src/common/gameState';
+import { CellType } from "../utils";
+
+const CELL_STROKE_COLOR = "white";
+const CELL_SIZE = 20;
+const DISTANCE_BETWEEN_HEXAGONE_PARALLEL_SIDES = Math.sqrt(3);
+
+interface BoardData {
+  gameState: GameState;
+}
+
+export default function Board(props: BoardData) {
+  return (
+    <Svg width={getApproximateSvgSize(props.gameState.board.length)} height={getApproximateSvgSize(props.gameState.board.length)}>
+      {
+        generateBoardCells(props.gameState).map((cell, index) => (
+          cell.cellType == "playable" ?
+            <PlayableCell
+              key={index}
+              svgPoints={cell.svgPointsToDraw}
+              strokeColor={CELL_STROKE_COLOR}
+              cellValue={props.gameState.board[cell.withoutBorderCoordinates.y][cell.withoutBorderCoordinates.x]}
+              onCellPress={() => onCellPress(cell.withoutBorderCoordinates)}
+            />
+            :
+            <BorderCell
+              key={index}
+              svgPoints={cell.svgPointsToDraw}
+              strokeColor={CELL_STROKE_COLOR}
+              playerBorder={cell.cellType}
+            />
+        ))

```

Comment:
IMO destructure properties is a good way to read faster what you have inside your object and what you use inside your function. It's easier to understand.


```suggestion
        generateBoardCells(props.gameState).map(({withoutBorderCoordinates, svgPointsToDraw, cellType}, index) => (
          cellType == "playable" ?
            <PlayableCell
              key={index}
              svgPoints={svgPointsToDraw}
              strokeColor={CELL_STROKE_COLOR}
              cellValue={props.gameState.board[withoutBorderCoordinates.y][withoutBorderCoordinates.x]}
              onCellPress={() => onCellPress(withoutBorderCoordinates)}
            />
            :
            <BorderCell
              key={index}
              svgPoints={svgPointsToDraw}
              strokeColor={CELL_STROKE_COLOR}
              playerBorder={cellType}
            />
        ))

```

Reply:
Oh, be careful, I code inside a comment. it could not works well !